### ISSUE TITLE
Image Component: esc button doesn't close the preview

### ISSUE THREAD

**Issue by GeorgeHulpoi** [primefaces/primeng#13305]
### Describe the bug

Image Component should be closed when you press on ESC button. Even on https://primeng.org/image doesn't work. 

### Environment

-

### Reproducer

https://primeng.org/image

### Angular version

16.0.2

### PrimeNG version

latest

### Build / Runtime

Angular CLI App

### Language

TypeScript

### Node version (for AoT issues node --version)

unknown

### Browser(s)

Edge, Chrome, to be confirmed on the rest

### Steps to reproduce the behavior

Click on the preview and try to press ESC.

### Expected behavior

It should close the Image preview

**Comment by SoyDiego** [comment]
Hi @GeorgeHulpoi, I have checked the component image and you are right.
I have create a PR https://github.com/primefaces/primeng/pull/13309 fixing that part.
If the PrimeNG Team approves, it will be in the next release.
